.github/workflows/run-cluster.yaml

```diff
@@ -3,9 +3,13 @@ name: run-cluster
 on:
   workflow_dispatch:
     inputs:
+      daft_wheel_url:
+        type: string
+        description: The url pointing to a daft python-wheel for which to install and run
```

Comment:
Specifically, a public https URL to the wheel directly right

Reply:
Yup, exactly.

Reply:
Will update the description in a follow-up PR.

Reply:
Updates here: https://github.com/Eventual-Inc/Daft/pull/3473.